fedora_commons/Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM tomcat:8.5
 
-# Download Fedora Commons 4.6, update default context file
+# Add Fedora Commons 4.6.1, update default context file, example data and wait-for-it script
 ADD context.xml /usr/local/tomcat/conf/context.xml
-
-RUN wget --progress=dot:giga https://github.com/fcrepo4/fcrepo4/releases/download/fcrepo-4.6.0/fcrepo-webapp-4.6.0.war -O /usr/local/tomcat/webapps/fcrepo.war
+ADD fcrepo-webapp-4.6.1.war /usr/local/tomcat/webapps/fcrepo.war
+ADD example_willow_objects.tar.gz /tmp/
+ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh /usr/local/bin/wait-for-it.sh
```

Comment:
Might be worth linking to a specific commit, not master. (Same as you would with fedora-4.6.1 rather than fedora-latest)